source/features/html-preview-link.tsx

```diff
@@ -25,7 +25,7 @@ function add(rawButton: HTMLAnchorElement): void {
 }
 
 function init(signal: AbortSignal): void {
-	observe('a:is(#raw-url, [data-testid="raw-button"])', add, {signal});
+	observe(['a#raw-url', 'a[data-testid="raw-button"])'], add, {signal});
```

Comment:
`:is` was being used with little benefit so I dropped it, however in the final version `is`/`has` selectors are excluded